packages/client/hmi-client/src/types/common.ts

```diff
@@ -19,6 +19,12 @@ export interface FeatureConfig {
 	isPreview: boolean;
 }
 
+export enum ParamType {
+	CONSTANT = 'constant',
+	DISTRIBUTION = 'distribution',
+	TIME_SERIES = 'time_series'
+}
+
```

Comment:
If you never need those strings, just do not add them, keeps the ENUM simplier.
```suggestion
export enum ParamType { CONSTANT, DISTRIBUTION, TIME_SERIES }
```